scrapers/urbandictionary.rb

```diff
@@ -21,6 +25,13 @@ def get_definitions(term)
     definitions.map { |m| parse_definition(term, m) }
   end
 
+  def get_random_definition()
```

Comment:
I know the others do it, but I think we should lose the `get_` at the start of the method, as it's implicit. (I think the rest should go, too).

We can also avoid having the `()` here, too.